apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx

```diff
@@ -49,256 +162,406 @@ export const AddressEntryForm = ({ action, entry, onChange, onCancel }: EntryPro
+                                            onKeyDown={handleKeyDown}
+                                            onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                                                onChange(e);
+                                                populateSuggestions(e);
+                                            }}
+                                            defaultValue={value}
+                                            type="text"
+                                            label="Street address 1"
+                                            name="address1"
+                                            htmlFor="address1"
+                                            id="address1"
+                                        />
+                                    )}
                                 />
-                            )}
-                        />
-                    </Grid>
-                    <Grid col={12} className="border-bottom border-base-lighter padding-bottom-2 padding-2">
-                        <Controller
-                            control={control}
-                            name="address2"
-                            defaultValue={entry.address2}
-                            render={({ field: { onChange, value } }) => (
-                                <Input
-                                    flexBox
-                                    onChange={onChange}
-                                    defaultValue={value}
-                                    type="text"
-                                    label="Street address 2"
+
+                                {showSuggestions && (
+                                    <div className="button-group">
+                                        <ul
+                                            ref={wrapperRef}
+                                            id="basic-nav-section-one"
+                                            className="suggestion-list-container usa-nav__submenu">
+                                            {suggestions.map((suggestion, idx) => (
+                                                <li key={idx} className="usa-nav__submenu-item">
+                                                    <SearchCriteriaContext.Consumer>
+                                                        {({ searchCriteria }) => {
+                                                            return (
+                                                                <Button
+                                                                    key={idx}
+                                                                    onClick={() =>
+                                                                        handleSuggestionSelection(
+                                                                            idx,
+                                                                            searchCriteria.states
```

Comment:
The state list is available in the component in `location.states`.  Trying to keep `SearchCriteriaContext` use limited in favor of the hooks that pull multiple lists at once.

Reply:
Sounds good.